### PR TITLE
Allow copying to clipboard in Electron

### PR DESCRIPTION
**Pull request by gzdunek**
Contributes to https://github.com/gravitational/teleport/issues/30422
This PR enables the `clipboard-sanitized-write` permission for the `BrowserWindow`.
This was suggested in [my previous PR](https://github.com/gravitational/teleport/pull/34929#discussion_r1406350364), and indeed, it is much more convenient to use than a separate IPC.

When it comes to security, exposing that API doesn't seem to have any negative consequences. The only difference I see is that the IPC allowed only text content, while the Clipboard API allows for any content. However, this content seems to be sanitized (and it is allowed in a regular browser anyway). 
The small advantage (comparing to the IPC) is that we get the browser protection for:
>The request to write to the clipboard must be triggered during a user gesture. A call to clipboard.write or clipboard.writeText outside the scope of a user gesture (such as "click" or "touch" event handlers) will result in the immediate rejection of the promise returned by the API call.

(I tested it and it works).

To increase the overall security, I followed the suggestion from Doyensec blog:
>If your application needs to allow the renderer process permission to access some web APIs, you can add exceptions by modifying the permission handler. We recommend checking if the origin requesting permission matches an expected origin.

To achieve that, I had to extract the window URL to be known upfront (so we can check if it maches `requestingUrl`).
I tested the change to `loadURL` on macOS, Windows and Ubuntu.

### CODE REVIEW

File: web/packages/design/src/utils/copyToClipboard.ts [hunk 1]
```diff
@@ -15,13 +15,19 @@
...
 /**
- * copyToClipboard copies text to clipboard.
+ * Copies text to clipboard.
  *
  * @param textToCopy the text to copy to clipboard
  */
-export default function copyToClipboard(textToCopy: string): Promise<any> {
-  return navigator.clipboard.writeText(textToCopy).catch(err => {
-    // This can happen if the user denies clipboard permissions:
-    window.prompt('Cannot copy to clipboard. Use ctrl/cmd + c', err);
```
Comment (gzdunek) [inline web/packages/design/src/utils/copyToClipboard.ts:25]: I think we wanted to use `window.alert` here (which works in Electron btw).

Comment (ravicious) [reply web/packages/design/src/utils/copyToClipboard.ts:25]: I think it was actually meant to be `window.prompt` with `textToCopy` as the second argument. This way if `navigator.clipboard.writeText` failed, the browser would show a prompt with the input prefilled. Then you can simply copy the contents, which might be already preselected.

Comment (gzdunek) [reply web/packages/design/src/utils/copyToClipboard.ts:25]: Makes sense, I didn't think of it! 
So for Web UI we will use `window.prompt` and for Connect `window.alert`.

**Review by rudream — APPROVED** [review]

**Review by zmb3 — APPROVED** [review]

**Review by ravicious — APPROVED** [review]
Just for posterity, the blog post from Doyensec: https://blog.doyensec.com/2022/09/27/electron-api-default-permissions.html

**Comment by gzdunek** [comment]
Since I already touched `copyToClibpobard`, I will I also convert to it to use a named export.

**Pull request merged**